<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineDimensionsTab</class>
 <widget class="QWidget" name="LineDimensionsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>P1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>_p1X</cstring>
       </property>
       <property name="si" stdset="0">
        <string>pos X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="_p2Y">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>P2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>_p2X</cstring>
       </property>
       <property name="si" stdset="0">
        <string>geo X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="_p1X">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="_p2X">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QCheckBox" name="_lockPosToData">
       <property name="text">
        <string>Lock Position To Data</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="_dimXlabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> , </string>
       </property>
       <property name="buddy">
        <cstring>_p2Y</cstring>
       </property>
       <property name="si" stdset="0">
        <string>geo Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> , </string>
       </property>
       <property name="buddy">
        <cstring>_p1Y</cstring>
       </property>
       <property name="si" stdset="0">
        <string>pos Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="_p1Y">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_lockPosToData</tabstop>
  <tabstop>_p1X</tabstop>
  <tabstop>_p1Y</tabstop>
  <tabstop>_p2X</tabstop>
  <tabstop>_p2Y</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_lockPosToData</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LineDimensionsTab</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_p1X</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LineDimensionsTab</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_p2X</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LineDimensionsTab</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_p1Y</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LineDimensionsTab</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_p2Y</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LineDimensionsTab</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modified()</slot>
 </slots>
</ui>
